inventories/example_local/group_vars/dojot-k8s/dojot.yaml

```diff
@@ -35,12 +39,21 @@ dojot_vernemq_replicas: 1
 # for Kafka topics
 dojot_bridges_replicas: 1
 
+# Enable Kafka WS TLS support, the service must have a configured volume with the certificates
+# inside
+dojot_kafka_ws_enable_tls: false
+# Enable Kong (apigw) TLS support, the service must have a configured volume with the certificates
+# inside
+dojot_apigw_enable_mutual_tls: false
+
 # Enables fixed nodeports
 dojot_fixed_nodeports_enabled: false
 
 # Fixed nodeports to use. It's used if dojot_fixed_nodeports_enabled is true
 dojot_nodeports:
-  apigw: 30001
+  apigw:
+    http: 30001
+    https: 30002
```

Comment:
Should this port be mapped in haproxy?

Reply:
Yes, it should be mapped to HTTPS default port (443).

Reply:
Ok, won't the mapping be treated in this PR?

Reply:
Yes, it will be treated, I didn't see this configuration.